presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/FunctionCallProvider.java

```diff
@@ -35,35 +40,47 @@
     private final Optional<WindowFrame> frame;
     private final boolean distinct;
     private final List<PlanTestSymbol> args;
+    private final List<PlanMatchPattern.Ordering> orderBy;
 
-    private FunctionCallProvider(boolean isWindowFunction, QualifiedName name, Optional<WindowFrame> frame, boolean distinct, List<PlanTestSymbol> args)
+    private FunctionCallProvider(boolean isWindowFunction, QualifiedName name, Optional<WindowFrame> frame, boolean distinct, List<PlanTestSymbol> args, List<PlanMatchPattern.Ordering> orderBy)
```

Comment:
It would be nice to have a `Builder` here, like com.facebook.presto.sql.planner.assertions.WindowMatcher.Builder

Reply:
Yup. I'll leave that alone for now, though. We may need some cleanup in a few places in this code.